nodes/src/nodes/ner/IInstance.py

```
@@ -30,87 +30,86 @@
                 # Group entities by type
                 entities_by_type = {}
                 for entity in entities:
                     entity_type = entity['entity_group']
                     if entity_type not in entities_by_type:
                         entities_by_type[entity_type] = []
                     entities_by_type[entity_type].append(entity['word'])
-                
+
                 # Add to metadata (deduplicate and sort)
                 for entity_type, words in entities_by_type.items():
                     unique_words = sorted(list(set(words)))
```

Ruff (0.15.7)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)